config/locales/en.yml

```diff
@@ -256,7 +256,7 @@ en:
           **NOTE:** *In order for Let's Encrypt to generate a TLS certificate, your Rancher server must be accessible via the Internet at the FQDN provided over TCP port 80 (http), and you must supply an email address for future management.*
       wrapup:
         title: "Congratulations - your Rancher server is ready!"
-        failed_title: "Oh no! Your Rancher deployment has failed!"
+        failed_title: "Oh no! Your Rancher resources clean up has failed!"
```

Comment:
The failed_title is used when a deployment fails. Overloading it for when cleanup fails is not going to work - as now when a deployment fails it will say "resources clean up has failed". These two issues need different messages.